pkgs/development/tools/omnisharp-roslyn/updater.sh

```diff
@@ -29,38 +18,5 @@ fi
 
 cd ../../../..
 update-source-version omnisharp-roslyn "${new_version//v}"
-store_src="$(nix-build . -A omnisharp-roslyn.src --no-out-link)"
-src="$(mktemp -d /tmp/omnisharp-roslyn-src.XXX)"
-
-cp -rT "$store_src" "$src"
-chmod -R +w "$src"
-trap 'rm -r "$src"' EXIT
-
-pushd "$src"
-
-export DOTNET_NOLOGO=1
-export DOTNET_CLI_TELEMETRY_OPTOUT=1
-
-mkdir ./nuget_pkgs
-
-replaceInPlace global.json '7.0.100-preview.4.22252.9' "$SDK7_VERSION"
-
-# This is only needed for restore as we'll build for the runtime that is compiling the code in the nix build.
-for project in src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj src/OmniSharp.LanguageServerProtocol/OmniSharp.LanguageServerProtocol.csproj; do
-    replaceInPlace $project \
-        '<RuntimeIdentifiers>win7-x64;win7-x86;win10-arm64</RuntimeIdentifiers>' \
-        '<RuntimeIdentifiers>linux-x64;linux-arm64;osx-x64;osx-arm64</RuntimeIdentifiers>'
-done
-
-for project in src/OmniSharp.Stdio.Driver/OmniSharp.Stdio.Driver.csproj; do
-  dotnet restore "$project" \
-    --packages ./nuget_pkgs \
-    -property:PackageVersion="${new_version//v}" \
-    -property:AssemblyVersion="${new_version//v}".0 \
-    -property:FileVersion="${new_version//v}".0 \
-    -property:InformationalVersion="${new_version//v}" \
-    -property:RuntimeFrameworkVersion=6.0.0-preview.7.21317.1 \
-    -property:RollForward=LatestMajor
-done
 
-nuget-to-nix ./nuget_pkgs > "$deps_file"
+$(nix-build -A omnisharp-roslyn.fetch-deps --no-out-link) "$deps_file"
```

Comment:
Not sure if this has been tested, but omnisharp has some .NET 7 dependencies (no idea why. although it can be compiled and ran on .NET 6, it cannot be restored on 7)

Reply:
Worked fine from my testing.

Reply:
Oh ok, guess that was some weird compatibility issue I had with all the changes I was making then